client/webserver/site/src/html/bodybuilder.tmpl

```diff
@@ -12,6 +12,15 @@
   <link href="/css/style.css?v={{commitHash}}" rel="stylesheet">
 </head>
 <body class="dark{{if .UseDEXBranding}} dex-branding{{end}}">
+  {{if and (ne .Version .LatestVersion) .LatestVersion}}
```

Comment:
What exactly is this comparison. Sorry.

if and is used for two comparisons, then those are,  .Version != .LatestVersion and .LatestVersion existing at all?

I still think you should parse the version and probably do this somewhere else. It should compare the versions properly. Will this make our testing builds always hit?